### PR TITLE
Altom week 2

### PR DESCRIPTION
**Pull request by AltomHussain**
# Your Details

Your Name:
Your City:
Your Slack Name:

# Homework Details

Module:
Week:

### CODE REVIEW

File: week-1/Homework/mandatory/1-syntax-errors.js [hunk 1]
```diff
@@ -1,32 +1,40 @@
...
+  return (
+    "Hello, my name is " + name + " " + "and I am " + age + " " + "years old"
+  );
+}
 function getRemainder(a, b) {
-    remainder = a %% b;
-
-    // Use string interpolation here
-    return "The remainder is %{remainder}"
+  remainder = a % b;
```
Comment (Abdoulrazack95) [inline week-1/Homework/mandatory/1-syntax-errors.js:13]: You haven't declared a variable keyword here.

File: week-2/Homework/extra/1-radio-stations.js [hunk 2]
```diff
@@ -25,8 +33,50 @@
...
+    105,
+    106,
+    107,
+    108,
+  ];
 
-
+  let stations = [];
+  for (let i = 0; i < AllFrequencies.length; i++) {
+    var isItRadioStation = isRadioStation(89);
```
Comment (Abdoulrazack95) [inline week-2/Homework/extra/1-radio-stations.js:64]: You already have isRadioStation function which checks if the frequency existe or not. Have a look to the solution and Understand if you have any question in this part feel free to ask.

File: week-2/Homework/extra/2-array-methods.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+//Create a function that takes an array and returns the sum of all numbers in the array.
+function getSumOfItems(arr) {
+  let sum = 0;
+  arr.forEach((k) => {
+    sum += k;
+  });
+  console.log(sum);
+}
+
+getSumOfItems([-2, 84, 23]); //Expected: 105
```
Comment (Abdoulrazack95) [inline week-2/Homework/extra/2-array-methods.js:56]: Good Work Altom

File: week-2/Homework/mandatory/1-fix-functions.js [hunk 4]
```diff
@@ -1,44 +1,69 @@
...
+console.log(mood());
+// 2) For any numerical value greater or equal to 10
+//    Hint: use constant `num` and only change isBigEnough.
+//    Variable isBigEnough needs to evaluate to a boolean
 
 function greaterThan10() {
-  let num = 10;
-  let isBigEnough;
+  const num = 10;
+  const isBigEnough = true;
```
Comment (Abdoulrazack95) [inline week-2/Homework/mandatory/1-fix-functions.js:27]: Here you have wrong, isBigEnough should isBigEnough = num >= 20 or 30; then the if else statement return the message if the isBigEnough is true or false based on the variable which true or false with the comparaison operator. Google true or false comparaison operator.

File: week-2/Homework/mandatory/2-function-creation.js [hunk 5]
```diff
@@ -5,8 +5,18 @@ Write a function that:
...
 */
-function tidyUpString(strArr) {}
-
+// let String = Arr.map((tidy) => tidy.trim().toLowerCase().replace("/", ""));
+
+// function under(name) {
+//   return name.trim().toLowerCase().replace("/", "");
+// }
+function tidyUpString(strArr) {
+  let String = strArr.map((key) => key.trim().toLowerCase().replace("/", ""));
```
Comment (Abdoulrazack95) [inline week-2/Homework/mandatory/2-function-creation.js:14]: String word was already taken change it to another name.

File: week-2/Homework/mandatory/3-playing-computer.js [hunk 6]
```diff
@@ -7,28 +13,29 @@
...
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  // printed to the values of f1(), f2(),value of a and the value of x 
+  4. How many times is "f1" called? // is called twice when it come to the the odd number 
+  5. How many times is "f2" called? // is called three times when it comes to the even number
+  6. What value does the "a" parameter take in the first "f1" call? // it takes 8
+  7. What is the value of the "a" outer variable when "f1" is called for the first time? // it is 7
 */
```
Comment (Abdoulrazack95) [inline week-2/Homework/mandatory/3-playing-computer.js:24]: You did well on fixing this exercise.

**Comment by AltomHussain** [comment]
Thank you for reviewing my work

**Pull request closed without merging**